openshift/templates/pims-api/deploy.yaml

```diff
@@ -140,26 +140,26 @@ objects:
                 requests:
                   cpu: "${CPU_REQUEST}"
                   memory: "${MEMORY_REQUEST}"
-              # readinessProbe:
-              #   httpGet:
-              #     path: /hc
-              #     port: ${{APP_PORT}}
-              #     scheme: HTTP
-              #   initialDelaySeconds: 10
-              #   timeoutSeconds: 1
-              #   periodSeconds: 10
-              #   successThreshold: 1
-              #   failureThreshold: 3
-              # livenessProbe:
-              #   httpGet:
-              #     path: /hc
-              #     port: ${{APP_PORT}}
-              #     scheme: HTTP
-              #   initialDelaySeconds: 10
-              #   timeoutSeconds: 1
-              #   periodSeconds: 10
-              #   successThreshold: 1
-              #   failureThreshold: 3
+              readinessProbe:
+                failureThreshold: 3
+                httpGet:
+                  path: /api/ready
+                  port: 8080
```

Comment:
Could you please change it to use `port: ${{APP_PORT}}` instead of hardcoding it to 8080 to keep template generic?

Please note the double braces -> `${{APP_PORT}}` 

This replaces with the "number" representation (e.g. port: 8080) vs the string value (port: "8080")